src/services/resource-service.ts

```diff
@@ -81,6 +112,16 @@ export const ResourceService = {
   ): Promise<AxiosResponse<ItemsWithCount<Question>>> => {
     return axiosClient.get(URLs.resources.questions.get, { params })
   },
+
+  getQuestionsQuery: (params?: GetResourcesParams) => {
+    return baseService.request<ItemsWithCount<Question>>({
+      method: 'GET',
+      url: getFullUrl({
+        pathname: URLs.resources.questions.get,
+        searchParameters: params
```

Comment:
Are you sure that it's working without destructuring?
```suggestion
        searchParameters: { ...params }
```